### PR TITLE
Add the possibility to use the trackers in OpenXR

### PR DESCRIPTION
**Pull request by S-Dafarra**
With this PR, I exploit the [XR_HTCX_vive_tracker_interaction](https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_HTCX_vive_tracker_interaction) extension to retrieve the HTC trackers' pose.

The required changes were unfortunately pretty big, as it required to rework the way I was selecting the set on inputs for OpenXR. 

I have tested with the latest SteamVR beta (1.22.11) and seems to be working fine. The SteamVR side is not very mature in any case. In order to make it work, I had to pair again the trackers for some reason.

### CODE REVIEW

**Comment by S-Dafarra** [comment]
Rebased on top of main to use the new ci version

File: src/devices/openxrheadset/OpenXrHeadset.cpp [hunk 1]
```diff
@@ -778,13 +654,22 @@ bool yarp::dev::OpenXrHeadset::getTouch(unsigned int /*touch_id*/, yarp::sig::Ve
     return false;
 }
 
-std::string yarp::dev::OpenXrHeadset::getInteractionProfile()
+std::string yarp::dev::OpenXrHeadset::getLeftHandInteractionProfile()
+{
+    yCTrace(OPENXRHEADSET);
```
Comment (traversaro) [inline src/devices/openxrheadset/OpenXrHeadset.cpp:659]: It this doing something?

**Review by traversaro — APPROVED** [review]
Minor comment.

File: src/devices/openxrheadset/OpenXrHeadset.cpp (hunk 1 above)
Comment (S-Dafarra) [reply src/devices/openxrheadset/OpenXrHeadset.cpp:659]: To be honest, this was something I "cargo-culted" from ``yarp-device-ovrheadset``. See for example https://github.com/robotology/yarp-device-ovrheadset/blob/b310d7c8588ed0c74d2d167ea41c7d804d10aa18/src/devices/ovrheadset/OVRHeadset.cpp#L271

I was thinking that it does some tracing, but I am not so sure

Comment (S-Dafarra) [reply src/devices/openxrheadset/OpenXrHeadset.cpp:659]: I will remove them all

Comment (traversaro) [reply src/devices/openxrheadset/OpenXrHeadset.cpp:659]: I am not sure, but I think `yCTrace(..)` is basically a stream object, so those `yCTrace(..)` in the constructor originally contained something like `yCTrace(..) << "this is a costructor"` or similar.

Comment (S-Dafarra) [reply src/devices/openxrheadset/OpenXrHeadset.cpp]: Removed in https://github.com/ami-iit/yarp-device-openxrheadset/pull/18/commits/582c44c7f24c41139b8efd367993d35713cdcdc0

**Comment by S-Dafarra** [comment]
Thanks @traversaro! Merging

**Pull request merged**